app/components/attachment_button.js

```diff
@@ -328,6 +330,20 @@ export default class AttachmentButton extends PureComponent {
         }, 100);
     };
 
+    removeProfileImage = () => {
+        this.props.removeProfileImage();
```

Comment:
the prop for removeProfile image can be null and you are not validating that here, potential crash